### ISSUE TITLE
Guide for beginners

### ISSUE THREAD

**Issue by Gagazet** [TensorBox/TensorBox#188]
Hello guys! Is anyone can explain, please, for new people at programming and,  in particular, at TensorBox how to use it step by step? I feel, like a lot of people will be glad to read guide and try to do their project's with it~ Maybe, for a lot of people, who already *sharks* at coding - that's easy, but there is newbie's too. Thank you for it~

**Comment by Gagazet** [comment]
Can someone explain process, please, how to work with your own data? With video, for example~

**Comment by Russell91** [comment]
We have a developer chat channel at https://gitter.im/Russell91/TensorBox. You might have some luck talking with people there. We should definitely put together an intro video at some point though. Would be interested to chat with others about that possibility.

**Comment by Vardan95** [comment]
@Russell91 What is the proper way to export trained model as a tensroflow graph and use it in a mobile app?

**Comment by mkreich** [comment]
Here is what I did:

1. frezze graph with freeze_graph.py. 
2. strip graph (optional) 
3. use protobyte Graph file in a JNI Interface the same way as in the tensorflow android demo app

**Comment by Vardan95** [comment]
where did you put your code to get .pb file?

**Comment by mkreich** [comment]
I copied stuff from build forward to initialise a session, then I used 
```
protobyte = session.graph.as_graph_def() 
with open(file, 'w') as f:  f.write(protobyte.SerializeToString())
```

**Comment by Vardan95** [comment]
And after that, you was succeeded to run your model on mobile?

**Comment by mkreich** [comment]
Yes.
The complete part of build forward and some essential the train method.
You need to "replicate" the steps for setting up a session, after that you can save this session like I have written above. 
I recommend writing an own python script for loading the checkpoints, which imports the essential stuff from tensorbox.

**Comment by Vardan95** [comment]
ok, thanks

**Comment by hareshnindblr76** [comment]
I'm trying to use Reinspect with resnet_v1_101. Can someone please explain the reason behind using just 832 of the 2048 features from output of block_4 as input to lstm?
And also the intuition behind using spatial avg_pooling on just the last 132 channels.
`    elif H['avg_pool_size'] > 1:
        pool_size = H['avg_pool_size']
        cnn1 = cnn[:, :, :, :700]
        cnn2 = cnn[:, :, :, 700:]
        cnn2 = tf.nn.avg_pool(cnn2, ksize=[1, pool_size, pool_size, 1],
                              strides=[1, 1, 1, 1], padding='SAME')
        cnn = tf_concat(3, [cnn1, cnn2])
`
